subsys/mgmt/fmfu/src/mgmt_fmfu.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include <mgmt/mgmt.h>
+#include <nrf_fmfu.h>
+#include <mgmt/mgmt_fmfu.h>
+#include "cborattr/cborattr.h"
+#include <stats/stats.h>
+
+LOG_MODULE_REGISTER(mgmt_fmfu, CONFIG_MGMT_FMFU_LOG_LEVEL);
+
+/* Define full modem update mgmt spesific SMP return codes */
```

Comment:
For all of these, IMO we should drop the `define`. Instead of stating what you do, just state what it is